bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/util/RowKeyUtil.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.google.cloud.bigtable.util;
+package com.google.cloud.bigtable.hbase.util;
```

Comment:
I don't think you can move this class. It's marked as InternalExtensionOnly. Instead, I think you have to copy it & add it your list of classes to deprecate in the future